### PR TITLE
Update reseed whitelist json

### PR DESCRIPTION
**Pull request by pizzaman1337**
Updates reseed whitelist json dynamically

### CODE REVIEW

File: protocol/reseed/dataConverts/convertWhitelist.js [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+
+    var finalOutput = [];
+
+    // loop through output and put into finalOutput in order of tokensToWhitelist
+    for (const token of tokensToWhitelist) {
+      finalOutput.push(output[token]);
+    }
+
+    fs.writeFileSync(outputFilePath, JSON.stringify(finalOutput, null, 2));
+    console.log("Field JSON has been written successfully");
```
Comment (nickkatsios) [inline protocol/reseed/dataConverts/convertWhitelist.js:237]: "Whitelist JSON..."

File: protocol/reseed/dataConverts/convertWhitelist.js [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+          tokenAssetSettings.selector,
+          tokenAssetSettings.stalkEarnedPerSeason,
+          tokenAssetSettings.stalkIssuedPerBdv,
+          tokenAssetSettings.milestoneSeason,
+          tokenAssetSettings.milestoneStem,
+          tokenAssetSettings.encodeType,
+          tokenAssetSettings.deltaStalkEarnedPerSeason,
+          tokenToGpAndOptimalPercentDepositedBdv[0],
+          tokenToGpAndOptimalPercentDepositedBdv[1],
+          liquidityWeightImpl, // these 2 are new so are set manually
```
Comment (nickkatsios) [inline protocol/reseed/dataConverts/convertWhitelist.js:217]: these should be the inverse. In `System.sol` the `AssetSettings` struct has the liquidity weight implementation last. 
Screenshot 2024-09-14 at 1 44 28 PM

**Review by nickkatsios — COMMENTED** [review]
Added a few comments, otherwise LGTM!

File: protocol/reseed/dataConverts/convertWhitelist.js (hunk 1 above)
Comment (Brean0) [reply protocol/reseed/dataConverts/convertWhitelist.js]: fixed here: 079bc90a720b09dc84ca3081695f667a778e6a77

File: protocol/reseed/dataConverts/convertWhitelist.js (hunk 2 above)
Comment (Brean0) [reply protocol/reseed/dataConverts/convertWhitelist.js]: fixed here: 7bb1ab646e2efddf9a9a88911ff1805879fb6906

**Pull request merged**